zingolib/src/lightclient/propose.rs

```diff
@@ -110,13 +115,17 @@ pub enum DoProposeError {
 }
 
 impl LightClient {
+    async fn store_proposal(&self, proposal: ZingoProposal) {
```

Comment:
I implemented this method yesterday or so...
called "update_proposal" I think...